gensim/corpora/wikicorpus.py

```diff
@@ -300,22 +305,26 @@ def get_texts(self):
         articles, articles_all = 0, 0
         positions, positions_all = 0, 0
         texts = ((text, self.lemmatize, title, pageid) for title, text, pageid in extract_pages(bz2.BZ2File(self.fname), self.filter_namespaces))
-        pool = multiprocessing.Pool(self.processes)
+        pool = multiprocessing.Pool(self.processes, init_worker)
```

Comment:
@piskvorky I was running `get_coherence` using `coherence="c_v"` and trying to program it to gracefully handle `KeyboardInterrupt`. You can see the mechanisms I put in place for this in the `text_analysis` module in the PR. While doing this, I faced some confusion because some background process was still raising `KeyboardInterrupt`. After some digging, I noticed that the `wikicorpus` pool workers were the culprit.

This pool may be active during many other phases of gensim execution if the underlying corpus/texts being iterated come from the `wikicorpus`. I think it makes things slightly cleaner in this case to handle the `KeyboardInterrupt` in some manner that does not propagate. Perhaps some sort of logging would improve this?